INTHEWILD.md

```diff
@@ -322,6 +322,7 @@ Currently, **officially** using Airflow:
 1. [Open Knowledge International](https://okfn.org) [@vitorbaptista](https://github.com/vitorbaptista)
 1. [Opensignal](https://www.opensignal.com) [@harrisjoseph](https://github.com/harrisjoseph)
 1. [OpenSlate](https://openslate.com) [@marcusianlevine](https://github.com/marcusianlevine)
+1. [Openverse](https://wordpress.org/openverse) 
```

Comment:
Thanks for the PR and being part of the Airflow community :)
There is a trailing whitespace that is failing the static checks for your PR:
 
```suggestion
1. [Openverse](https://wordpress.org/openverse)
```